### PR TITLE
add bastion tags option

### PR DESCRIPTION
**Pull request by amymaraisane7f**
only one variable added - bastion tags which is an array

### CODE REVIEW

**Review by chris-malloy — APPROVED** [review]

File: ec2.tf [hunk 1]
```diff
@@ -25,6 +25,10 @@ sudo yum -y update
 EOF
 }
 
+locals {
+  tags = var.bastion_tags
+}
+
```
Comment (jduv) [inline ec2.tf:31]: You don't need the locals here, just set the `for-each` equal to the variable you created. Comment below.

File: ec2.tf [hunk 2]
```diff
@@ -39,4 +43,12 @@ resource "aws_instance" "bastion_hosts" {
   tags = {
     Name = "Bastion Host ${count.index + 1}"
   }
+
+  dynamic "tag" {
+    for_each = local.tags
```
Comment (jduv) [inline ec2.tf:48]: Replace `local.tags` with `var.bastion_tags` and we're gtg.

File: variables.tf [hunk 3]
```diff
@@ -67,3 +67,12 @@ variable bastion_instance_type {
 variable bastion_key_name {
   description = "The key name for the bastion host without.pem on the end. Make sure you have access to it."
 }
+
+variable "bastion_tags" {
+  default     = []
```
Comment (jduv) [inline variables.tf:72]: Yay! Sane default!

**Review by jduv — CHANGES_REQUESTED** [review]

**Review by chelseagreen — DISMISSED** [review]

**Comment by jduv** [comment]
This is good. The tags block looks like every other tags block (responding to a comment from @HJain01  on slack.

Screen Shot 2021-09-07 at 09 01 47

Ship it.

**Review by jduv — APPROVED** [review]

**Review by chelseagreen — APPROVED** [review]

File: ec2.tf [hunk 4]
```diff
@@ -25,18 +25,27 @@ sudo yum -y update
...
+  }]
+}
+
 resource "aws_instance" "bastion_hosts" {
   count         = var.bastion_count
   ami           = data.aws_ami.ec2_linux.id
   instance_type = var.bastion_instance_type
   subnet_id     = aws_subnet.utility_subnet.id
   key_name      = var.bastion_key_name
+  tags          = concat(local.customTags, local.regularTags)
```
Comment (chris-malloy) [inline ec2.tf:46]: ❤️

**Review by chris-malloy — APPROVED** [review]

**Review by HJain01 — APPROVED** [review]

**Pull request merged**